llvm/docs/ReleaseNotes.rst

```diff
@@ -62,6 +62,9 @@ Changes to LLVM infrastructure
 Changes to building LLVM
 ------------------------
 
+* LLVM now supports rpmalloc in-tree, as a replacement C allocator for hosted
+  toolchains. Currently using version 1.4.5.
```

Comment:
Maybe this could be expanded a bit, with a link to the relevant section in CMake.rst, mentioning that the relevant flag is LLVM_ENABLE_RPMALLOC, that the main benefit is for LTO on Windows etc?

Reply:
As suggested.